src/main/java/Commands/Command.java

```diff
@@ -0,0 +1,30 @@
+package Commands;
+
+import Exceptions.DukeException;
+
+/**
+ * An interface skeleton for all UserCommands
+ */
+public interface Command {
+    /**
+     * Replies User when command is executed
+     *
+     * @throws DukeException the duke exception
+     */
+    public void reply() throws DukeException;
+
+    /**
+     * Executes the Action of the Commands.Command
+     *
+     * @return the boolean
+     * @throws DukeException the duke exception
+     */
+    public boolean action() throws DukeException, DukeException;
```

Comment:
Possible repetition of `DukeException`

Reply:
yes it is.. i saw it in my code i didnt push it